go-controller/pkg/ovn/libovsdbops/util.go

```diff
@@ -3,11 +3,27 @@ package libovsdbops
 import (
 	"context"
 	"fmt"
+	"math/rand"
+	"sync/atomic"
 
 	"github.com/ovn-org/libovsdb/client"
 	"github.com/ovn-org/libovsdb/ovsdb"
 )
 
+const (
+	namedUUIDPrefix = 'u'
+)
+
+var namedUUIDCounter = rand.Uint32()
+
+func IsNamedUUID(id string) bool {
+	return id != "" && id[0] == namedUUIDPrefix
+}
+
+func BuildNamedUUID() string {
```

Comment:
What is this used for? Can you add a docblock?

It looks like it builds sequential UUIDs? Why?

Techncially, UUIDs are supposed to encode their version in some of the middle bits. Does this generate conformant UUIDs?

Reply:
One more thing - the counter will, of course, reset every time ovn-kube restarts. What is the risk of collision?

Reply:
I will add doc. Named UUIDs are not UUIDs, they are temporary IDs that are only meaningful within the context of a single transaction.They only  need to be unique within that context so a global monotonically increasing context should be enough. See `<named-uuid>` in the 5.1 section of ovsdb rfc https://datatracker.ietf.org/doc/html/rfc7047.txt

Reply:
For a collision to happen, max<uint32> different named-uuids need to be built (with `BuildNamedUUID`), included and referenced in a single transaction.